packages/js/src/utils/id-token.ts

```diff
@@ -15,6 +20,18 @@ const IdTokenClaimsSchema = s.type({
 
 export type IdTokenClaims = s.Infer<typeof IdTokenClaimsSchema>;
 
+export const verifyIdToken = async (
+  idToken: string,
+  clientId: string,
+  issuer: string,
+  jwks: JWTVerifyGetKey
+) => {
+  const result = await jwtVerify(idToken, jwks, { audience: clientId, issuer });
```

Comment:
does `jwtVerify()` check expiration?

Reply:
Yes, tested.

Reply:
Related test case 'expired ID Token should throw'.